devito/mpi/distributed.py

```diff
@@ -375,6 +375,9 @@ def _obj_neighborhood(self):
         """
         return MPINeighborhood(self.neighborhood)
 
+    def create_new(self, shape, dimensions, comm):
```

Comment:
should be `_rebuild`

Reply:
also, the parameters should be kwargs such that you reuse self's if not provided

```
def rebuild(self, shape=None, ...):
    return Distributor(shape or self.shape, ...)
```